build.gradle

```diff
@@ -54,14 +54,14 @@ jar {
 }
 
 dependencies {
-    compile gradleApi()
-    compile deps.gson
+    implementation localGroovy()
```

Comment:
I've used the java-library plugin in thrifty, but it never occurred to me to use it here.  Thanks for updating!

Also, why did you remove `gradleApi()` - is that included somehow via `localGroovy()`?